app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -120,13 +121,52 @@ <h5>{% trans 'Payout Preview' %}</h5>
                     </a>
                   </div>
                 </div>
+                
+              </div>
+              <div id="tooltiphov">
+                    <span id="heading">Where is my Eth Going?</span><br>
+                    <span id="subheading">Your funds are safe!</span><br><br>
+
+                    <span id="hovertext"><strong>Unverified Users</strong> <em>(no eth address associated)</em><br>
+                    Gitcoin will create a proxy account and hold the funds there until the recipient claims the funds.<br><br>
+
+                    <strong>Verified Users</strong><br>
+                    We send the funds directly to the address associated with the user.</span>
               </div>
             </div>
           </div>
         </div>
       </div>
     </div>
     <style>
+      #tooltip {
```

Comment:
Can we move this CSS to either the top of the template or possibly a separate .css file for bounties specifically?

Reply:
`bounty.css` exists but I can't find that it is included anywhere.

Reply:
It's not included in this template, but the majority of our common CSS styles are included via:  `{% include 'shared/head.html' %}` cc @thelostone-mc @SaptakS What path would you like taken here?  Do you want the styles left inline or moved to another asset?

Reply:
Hmm right now `gitcoin.css` should be alright. that's where I've thrown in the autocomplete css code too